xdsl/dialects/scf.py

```diff
@@ -18,12 +26,17 @@ class If(Operation):
     false_region: Region
 
     @staticmethod
-    def get(cond: SSAValue | Operation, return_types: List[Attribute],
-            true_region: Region | List[Block] | List[Operation],
-            false_region: Region | List[Block] | List[Operation]) -> If:
-        return If.build(operands=[cond],
-                        result_types=[return_types],
-                        regions=[true_region, false_region])
+    def get(
+        cond: SSAValue | Operation,
+        return_types: List[Attribute],
+        true_region: Region | List[Block] | List[Operation],
+        false_region: Region | List[Block] | List[Operation],
```

Comment:
nit:
```suggestion
        false_region: Region | List[Block] | List[Operation]
```
We tend not to put trailing commas in the argument list either